client/src/components/menubar/index.tsx

```diff
@@ -110,154 +160,95 @@ class MenuBar extends React.PureComponent<MenuBarProps, State> {
     return key >= 0 && key <= 9;
   }
 
-  constructor(props: MenuBarProps) {
-    super(props);
-    const { layoutChoice, dispatch } = this.props;
-    this.state = {
-      pendingClipPercentiles: {
-        clipPercentileMin: undefined,
-        clipPercentileMax: undefined,
-      },
-    };
-    const currentConditionMet = layoutChoice?.current?.includes(
-      globals.spatialEmbeddingKeyword
-    );
-    // (seve): On some datasets, the app initially loads with a different layout selected, then switches to spatial.
-    //  This triggers the componentDidUpdate, which toggles the image underlay.
-    //  Other datasets correctly load with the spatial layout initially selected, but then don't trigger the componentDidUpdate.
-    if (currentConditionMet) {
-      dispatch({
-        type: "toggle image underlay",
-        toggle: true,
-      });
-    }
-  }
-
-  componentDidUpdate(prevProps: MenuBarProps): void {
-    const { layoutChoice, dispatch } = this.props;
-    const prevConditionMet = prevProps.layoutChoice?.current?.includes(
-      globals.spatialEmbeddingKeyword
-    );
-    const currentConditionMet = layoutChoice?.current?.includes(
-      globals.spatialEmbeddingKeyword
-    );
-
-    if (!prevConditionMet && currentConditionMet) {
-      dispatch({
-        type: "toggle image underlay",
-        toggle: true,
-      });
-    }
-  }
```

Comment:
These are not ported to the refactored functional component. We don't want to automatically toggle the underlay on when the embedding changes